Exercise_1/day1/Program10.java

```diff
@@ -0,0 +1,29 @@
+package day1;
+
+public class Program10 {
+	public static void main(String[] args) {
+
+		  int[][] array = { {1,2,3,4},
+				            {5,6,7,8},
+				            {9,10,11,12},
+				            {13,14,15,16}
+				           };
+		  System.out.println("Two Dimensional Matrix :");
+		  print(array);
+
+	  }
+	  public static void print(int array[][])
+	  {
+
```

Comment:
please remove extra empty lines in files and keep the intendation same for every files

Reply:
updated